static/js/chart/draw.ts

```diff
@@ -427,10 +476,253 @@ function drawLineChart(
   }
 }
 
+interface PlotParams {
+  colors: string[];
+  dashes: string[];
+  statVars: string[];
+  geoIds: string[];
+}
+
+/**
+ * Return all styles information for given dataGroupsDict.
+ * {
```

Comment:
this documentation now belongs with the interface.

Reply:
Done